lambdas/handlers/logout_handler.py

```diff
@@ -29,7 +29,7 @@ def logout_handler(token):
         ssm_response = get_ssm_parameter(key=ssm_public_key_parameter_name)
         jwt_class = jwt
         public_key = ssm_response["Parameter"]["Value"]
-        logger.info("decoding token")
+        logger.info(f"decoding token")
```

Comment:
Did you run the formatter here?

Reply:
Ah yes, I likely left this in after debugging. Good spot!